lib/EpdFont/scripts/fontconvert.py

```
@@ -275,24 +281,24 @@ def load_glyph(code_point):
         if is2Bit:
+            # Build out 4-bit greyscale bitmap
+            pixels4g = []
+            px = 0
+            for i, v in enumerate(bitmap.buffer):
+                y = i / bitmap.width
+                x = i % bitmap.width
+                if x % 2 == 0:
+                    px = (v >> 4)
+                else:
+                    px = px | (v & 0xF0)
+                    pixels4g.append(px);
```

Ruff (0.15.18)

[error] Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)